resolvers/webpack/.babelrc

```diff
@@ -1 +1 @@
-../../.babelrc
\ No newline at end of file
+../../.babelrc
```

Comment:
this was a symlink; why did this file change?

Reply:
this file import was failing on ApVeillor in case of node v10. 
I try to replace its content with 
```
{
  "extends": "../../.babelrc"
}
```
to avoid symling problem, but still present, so I rollback it.

I think the extension is a better way as the symlink for local development

Reply:
I don't know how to commit a sYmlinked file. can you do it after the merge ?